tests/foreman/ui/test_ansible.py

```diff
@@ -899,7 +898,7 @@ def _finalize():
         nc.update(['organization', 'location'])
         target_sat.api.AnsibleRoles().sync(data={'proxy_id': nc.id, 'role_names': SELECTED_ROLE})
         vm_hostnames = []
-        for vm in registered_hosts:
```

Comment:
We are already registering the hosts in rex_contenthosts fixture and then again later in tests.

Reply:
Good catch!! I'll update this too

Reply:
Done